src/App.js

```diff
@@ -9,8 +9,8 @@ from 'react-router-dom'
 import Header from './components/Header'
 import { Footer } from './components/Footer'
 import Home from './pages/Home'
-import FormSignUp from './pages/FormSignUp'
-import FormSignIn from './pages/FormSignIn'
+import SignUp from './pages/SignUp'
+import SignIn from './pages/SignIn'
```

Comment:
El page `./page/SigIn` no existe. te falto incluirlo

Reply:
Si está, 
![image](https://user-images.githubusercontent.com/65572735/100004820-09f00000-2d96-11eb-857a-0afda9a5e392.png)

Reply:
No se había cargado los cambios de sign in, ya quedo full